packages/api/src/controllers/asset.ts

```diff
@@ -47,7 +47,10 @@ const app = Router();
 
 const META_MAX_SIZE = 1024;
 
-function validateAssetMeta(meta: Record<string, string>) {
+function validateAssetMeta( meta: Record<string, string>, asset?:WithID<Asset> ) {
```

Comment:
Apparently the prettier is still not working on your env. Run `yarn prettier` manually from the project root instead.